projects/aca-content/src/lib/ui/overrides/adf-about.theme.scss

```diff
@@ -3,6 +3,13 @@
   adf-about {
     /* custom ADF About Component Theme */
 
+    .mdc-data-table__header-cell,
+    .mat-header-cell {
```

Comment:
I think `.mat-header-cell` was changed to `.mat-mdc-header-cell` so it is not used?